src/mds/Locker.cc

```diff
@@ -2609,6 +2609,12 @@ void Locker::mark_need_snapflush_inode(CInode *in)
   }
 }
 
+bool Locker::caps_reqd_by_other_client(client_t client)
+{
+  auto rlist = mds->locker->revoking_caps_by_client;
```

Comment:
"mds->locker-> " is unnecessary.  besides, the function name is not good. how about is_revoking_any_caps_from